components/Signup.js

```diff
@@ -133,6 +153,27 @@ const Signup = () => {
             <p className="form-error">{errors.password}</p>
           ) : null}
         </div>
+        <div className="custom-select">
+          <select
+            style={{ height: "5vh", width: "100%", borderRadius: "4px" }}
```

Comment:
move inline style to a css module https://beta.nextjs.org/docs/styling/css-modules

Reply:
Addressed